qiskit_machine_learning/kernels/fidelity_statevector_kernel.py

```diff
@@ -0,0 +1,135 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022, 2023.
```

Comment:
```suggestion
# (C) Copyright IBM 2023.
```

Reply:
Just a minor comment here - the PR was created last year and I think these files at the same time. So technically 2022 would be the first year of copyright if my statement hold true.